essay.tex

```diff
@@ -1,45 +1,97 @@
+\date{2017-11-16}
+\author{1604281}
 
 \begin{document}
 
 \maketitle
+\pagenumbering{arabic}
 
-\abstract{Please include an abstract of at most 100 words (these do not count towards your word count).}
+\begin{abstract}
+	The Agile Software Development methodology was initially designed to be used in small teams that could quickly develop products with flexible requirements. As Agile practices have gained popularity in the software industry, companies have found the need to employ Agile methods in larger teams and on more complex projects. This paper will be examining some of the challenges associated with up-scaling the Agile methodology and discussing some of the solutions that are used in industry.
+\end{abstract}
 
 \section{Introduction}
 
-Write your introduction here. A brief introduction is recommended, which should outline key details of the chosen topic and the reviewed papers, motivate the work, and provide a roadmap of key points to the reader. The motivation is quite important here, as essays should have a contribution (i.e., what is the point of the essay, and what does the reader take away from the essay) and the link between the motivation (in the introduction) and the contribution (in the conclusion) should be made clear.
+ One of the most common criticisms of the agile methodology is the difficulties encountered when up-scaling the size of Agile projects. \cite{begel2007usage}
+
+Although this paper is addressing general Agile practices and problems, the Agile methodology is also used in the games industry \cite{politowski2016old} which can range from small independent studios to massive teams working on multiple pieces of software. It has even been shown that many games development companies who do not intent to use Agile development still implement many of its core practices \cite{petrillo2010agility} and as such, the potential to scale Agile development to suit the nature of the game development environment could be beneficial to the games industry as a whole. 
+
+\section{Challenges}
+
+Though there are many issues involved in scaling an Agile project \cite{turk2014limitations}, the two that will be addressed in this paper are Team Size and Project Complexity. 
+
+\paragraph{Project Complexity}\mbox{}\newline
+
+As a project becomes larger and more complex, several issues arise when approaching it with the Agile methodology. Firstly, the Agile principle of ``Welcome changing requirements, even late in development." \cite{AgileManifesto}, while feasible to implement in a small project, becomes increasingly difficult as the software becomes more complex. For example, if a project has many constituent parts which each have many dependencies on each other, changing one aspect of the project late in development may cause problems in other parts of the project. This can be compounded by another aspect of project complexity - mixing of Agile and Non-Agile teams. \cite{reifer2003scaling}
```

Comment:
Put the `\cite{reifer2003scaling}` inside the sentence. e.g: `and Non-Agile teams \cite{reifer2003scaling}.`